tensorflow_datasets/core/lazy_imports.py

```diff
@@ -25,88 +25,79 @@
 
 
 def _try_import(module_name):
-  """Try importing a module, with an informative error message on failure."""
-  try:
-    mod = importlib.import_module(module_name)
-    return mod
-  except ImportError:
-    err_msg = ("Tried importing %s but failed. See setup.py extras_require. "
-               "The dataset you are trying to use may have additional "
-               "dependencies.")
-    utils.reraise(err_msg)
+    """Try importing a module, with an informative error message on failure."""
```

Comment:
Please be careful about indentation.

Reply:
Thanks for your review and patience! My code editor(VS code) was auto formatting  the code.